pkg/kobject/kobject.go

```diff
@@ -92,6 +92,7 @@ type ServiceConfig struct {
 	MemLimit        yaml.MemStringorInt `compose:"mem_limit" bundle:""`
 	TmpFs           []string            `compose:"tmpfs" bundle:""`
 	Dockerfile      string              `compose:"dockerfile" bundle:""`
+	Replicas        int                 `compose:"replicas" bundle:""`
```

Comment:
Actually... this would be more difficult than it's worth. Ignore me! Leave this as it is :+1: Just add a test for testing `replicas` within deploy.